tests/Feature/Console/Stack/StackLaravelCommandTest.php

```
@@ -0,0 +1,1566 @@
+use App\Commands\Stack\LaravelCommand;
+use App\Contracts\DockerExecutorInterface;
+use App\Contracts\InfrastructureManagerInterface;
+use App\Services\Stack\Installers\LaravelStackInstaller;
+use App\Services\Stack\StackLoaderService;
+use App\Services\Stack\StackRegistryManagerService;
+use LaravelZero\Framework\Commands\Command;
+
+/**
+ * Helper to create a test Laravel project directory.
+ */
+function createLaravelTestProject(string $projectName = 'test-project'): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$projectName'. (undefined)

(UnusedFormalParameter)